src/rollup/display-conditions/entity-fields/replace-entity-field.js

```diff
@@ -55,9 +55,18 @@ export default function replaceEntityField (
     // check for subfields if the value is an object
     var split = parts[0].trim().split('.');
 
-    dataval = window.lio.data;
-    var s;
+    // get entity data from tag
+    var dataval;
+    if (window.lio && window.lio.data) {
+      dataval = window.lio.data;
+    } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
+      var entity = window.jstag.getEntity();
+      if (entity && entity.data && entity.data.user) {
+        dataval = entity.data.user;
+      }
+    }
```

Comment:
notta big deal, but perhaps some of these repeated things sounds be functions. Like we could call a local `getEntity` function or something that handles the different tags.

Reply:
i actually want(ed) to refactor that entire file, its pretty cringy in spots but might be better as its own PR? opened this https://github.com/lytics/pathforajs/issues/316